orttraining/orttraining/eager/test/ort_ops.py

```diff
@@ -353,95 +347,84 @@ def test_resize(self):
-        assert torch.equal(cpu_out_tensor, ort_out_tensor.cpu())
-        assert torch.equal(ort_result.cpu(), ort_out_tensor.cpu())
-
-    def test_eq_tensor(self):
+    @parameterized.expand(math_sign_ops, name_func=rename_func)
+    def test_op_tensor(self, math_sign_ops):
         device = self.get_device()
-        cpu_a = torch.Tensor([1.0, 1.5, 2.0])
+        cpu_a = torch.Tensor([1.0, 1.5, 2.0, 3.5])
         ort_a = cpu_a.to(device)
-        cpu_b = torch.Tensor([1.0, 1.5, 2.1])
+        cpu_b = torch.Tensor([1.0, 1.4, 2.1, 2.4])
         ort_b = cpu_b.to(device)
 
         for tensor_type in {torch.float, torch.bool}:
-            for func in {"eq", "ne"}:
-                print(f"Testing {func} with type {tensor_type}")
-                cpu_out_tensor = torch.tensor([], dtype=tensor_type)
-                ort_out_tensor = cpu_out_tensor.to(device)
-                cpu_a_b_eq_result = eval(
-                    compile("torch." + func + "(cpu_a, cpu_b, out=cpu_out_tensor)", "<string>", "eval")
-                )
-                ort_a_b_eq_result = eval(
-                    compile("torch." + func + "(ort_a, ort_b, out=ort_out_tensor)", "<string>", "eval")
-                )
-                assert torch.equal(cpu_a_b_eq_result.to(device), ort_a_b_eq_result)
-                assert torch.equal(cpu_out_tensor, ort_out_tensor.to("cpu"))
-                assert ort_out_tensor.dtype == tensor_type
-
-    def test_eq_scalar(self):
+            print(f"Testing {math_sign_ops} with type {tensor_type}")
+            cpu_out_tensor = torch.tensor([], dtype=tensor_type)
+            ort_out_tensor = cpu_out_tensor.to(device)
+            cpu_a_b_result = eval(
+                compile("torch." + math_sign_ops + "(cpu_a, cpu_b, out=cpu_out_tensor)", "<string>", "eval")
+            )
+            ort_a_b_result = eval(
+                compile("torch." + math_sign_ops + "(ort_a, ort_b, out=ort_out_tensor)", "<string>", "eval")
+            )
+            assert torch.equal(cpu_a_b_result.to(device), ort_a_b_result)
+            assert torch.equal(cpu_out_tensor, ort_out_tensor.to("cpu"))
+            assert ort_out_tensor.dtype == tensor_type
+
+    @parameterized.expand(math_sign_ops, name_func=rename_func)
+    def test_op_scalar(self, math_sign_ops):
         device = self.get_device()
         cpu_tensor_int = torch.tensor([1, 1], dtype=torch.int32)
-        cpu_scalar_int = torch.scalar_tensor(1, dtype=torch.int)
-        cpu_scalar_int_not = torch.scalar_tensor(2, dtype=torch.int)
+        cpu_scalar_int_lt = torch.scalar_tensor(2, dtype=torch.int)
+        cpu_scalar_int_gt = torch.scalar_tensor(0, dtype=torch.int)
         cpu_tensor_float = torch.tensor([1.1, 1.1], dtype=torch.float32)
-        cpu_scalar_float = torch.scalar_tensor(1.1, dtype=torch.float32)
-        cpu_scalar_float_not = torch.scalar_tensor(1.0, dtype=torch.float32)
+        cpu_scalar_float_lt = 1.0  # torch.scalar_tensor(1.0, dtype=torch.float32)
```

Comment:
maybe rename `cpu_scalar_float_lt` to just `float_lt`, and the same for gt. Then delete `ort_scalar_float_lt` and gt.